homeassistant/components/filesize/sensor.py

```diff
@@ -5,21 +5,31 @@
 
 import voluptuous as vol
 
+from homeassistant.components.filesize.config_flow import UNIT_OF_MEASUREMENTS
 from homeassistant.components.sensor import PLATFORM_SCHEMA, SensorEntity
-from homeassistant.const import DATA_MEGABYTES
-import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    CONF_FILE_PATH,
+    CONF_UNIT_OF_MEASUREMENT,
+    DATA_BYTES,
+    DATA_MEGABYTES,
+)
 from homeassistant.helpers.reload import setup_reload_service
+from homeassistant.util import slugify
+import homeassistant.util.data_size as data_size
 
-from . import DOMAIN, PLATFORMS
+from .const import DOMAIN, PLATFORMS
 
 _LOGGER = logging.getLogger(__name__)
 
-
-CONF_FILE_PATHS = "file_paths"
 ICON = "mdi:file"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Required(CONF_FILE_PATHS): vol.All(cv.ensure_list, [cv.isfile])}
+    {
+        vol.Required(CONF_FILE_PATH): str,
```

Comment:
I don't think that we should change the YAML schema, as it will eventually be removed anyway. So instead, keep the existing structure, then import it to a config entry instead.